docs/explanation/reference-architecture.md

```diff
@@ -0,0 +1,45 @@
+
+Multipass is a tool to generate cloud-style Ubuntu VMs quickly on Linux, macOS and Windows. It provides a simple but powerful CLI that enables you to quickly access an Ubuntu command line or create your own local mini-cloud.
+
+```{figure} /images/multipass-reference-architecture.jpg
+   :alt: Multipass reference architecture diagram
+```
+
+## Clients
+
+A term we borrow from the client/server model. Multipass clients provide users with an interface to Multipass and its instances, delegating requests on the Multipass daemon. There are two clients in Multipass, both offering a user interface: CLI and GUI.
+
+## CLI (Command Line Interface)
+
+Is one of the two user interfaces that Multipass offers, in a client program that communicates with the daemon and which can be invoked on the command line: multipass. It is through this client that users can control Multipass and its instances in the terminal, with commands like multipass launch or multipass startstart.
+
+## GUI (Graphical User Interface)
+
+The Multipass GUI provides a visual, point-and-click interface for managing Multipass and its instances. It runs as a separate client application that communicates with, and delegates operations to, the Multipass daemon. The Multipass GUI provides a visual, point-and-click interface for managing Multipass and its instances. It runs as a separate client application that communicates with, and delegates operations to, the Multipass daemon.
+
+## Daemon/Service
+
+A long living process responsible for a variety of tasks; managing instances, client authentication, storing configuration settings, and providing file system shares. The daemon runs as a privileged user allowing it to interact directly with system resources and control which users have access to Multipass’s instances.
+
+## Storage/Mounts
+
+A filesystem share between the host machine and guest instance. Mounts are provisioned through SSH File System (SSHFS) and accurately translate ownership and user permissions between the host OS and the Linux guest.
+Instances - A Linux virtual machine hosted by the user’s machine. Multipass uses a hypervisor technology specific to the user’s native operating system which is used to emulate the instance.
+
+## Image Hosts
+
+Source to obtain disk images for the virtual machines. As of now, Multipass only supports communicating with image hosts that support Simplestreams. Multipass periodically fetches and updates the image metadata from image hosts and downloads virtual machine images on demand.
+
+## Networking
+
+Multipass creates a local network for virtual machines to communicate. This network has a random private subnet assigned to it, and each virtual machine would obtain an IP address via DHCP from this network by default. The virtual machines are accessible to each other through the private network. This virtual network is bridged to the primary host network by default to provide web connectivity to the VMs (meaning all the VMs are behind a NAT).
```

Comment:
```suggestion
Multipass creates a local network for virtual machines to communicate. This private network uses a dedicated subnet that each instance obtains an IP address from, via DHCP. Instances can be reached from the host and can communicate with each other through the private network, but they cannot be reached from the outside. Instances can also be bridged to the same network that the host is on, so that they can be reached from other participants (e.g. another computer in the user's home network).
```